### PR TITLE
Microsoft button added

### PR DESCRIPTION
**Pull request by champ96k**
new Microsoft button added

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -91,6 +91,14 @@ class SignInPage extends StatelessWidget {
             },
           ),
           Divider(),
+          SignInButton(
+            Buttons.Twitter,
+            text: "Microsoft ",
```
Comment (ZaynJarvis) [inline example/lib/main.dart:96]: example might need to be changed

**Comment by ZaynJarvis** [comment]
if possible, kindly provide a screenshot of the new button with ms.

File: example/lib/main.dart (hunk 1 above)
Comment (champ96k) [reply example/lib/main.dart:96]: okay sure

**Comment by champ96k** [comment]
> if possible, kindly provide a screenshot of the new button with ms.

here is new Microsoft button screenshot  [](https://i.ibb.co/8Ks23pd/Whats-App-Image-2020-08-09-at-9-48-54-PM.jpg)

**Comment by somarkoe** [comment]
For this, shouldn't an image be used (like Google's) in order to get the proper color scheme with their icon?

**Comment by champ96k** [comment]
> For this, shouldn't an image be used (like Google's) in order to get the proper color scheme with their icon?

this is not image, this is only icon and color can be changed as per requirement

**Comment by ZaynJarvis** [comment]
LGTM, will publish in next version

**Pull request merged**